sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/math.scala

```diff
@@ -520,3 +522,127 @@ case class Logarithm(left: Expression, right: Expression)
     """
   }
 }
+
+case class Round(child: Expression, scale: Expression)
```

Comment:
i'd add scaladoc for the expression to define the semantics of child and scale. also make sure you specify that scale must be a literal by the time execution happens.